mtags/src/main/scala-3/scala/meta/internal/pc/printer/MetalsPrinter.scala

```diff
@@ -255,7 +280,18 @@ class MetalsPrinter(names: ShortenedNames, dotcPrinter: DotcPrinter)(using
       // print only type string
       // e.g. "using Ord[T]" instead of "using x$0: Ord[T]"
       paramTypeString
-    else s"${keywordName}: ${paramTypeString}"
+    else
+      val isDefaultParam = param.isAllOf(DefaultParameter)
+      val default =
+        if includeDefaultParam && isDefaultParam then
+          val defaultValue = defaultValues.lift(index) match
+            case Some(value) if includeDefaultParam && !value.isEmpty => value
+            case _ => "{}"
```

Comment:
I wonder if this would be visually better:
```suggestion
            case _ => "..."
```

Reply:
I personally prefer `...` because `{}` seems like there's actually a default argument `{}`.

Reply:
Yeah, it was actually causing me to think it's an error. I will change that, since it's a small fix.